.github/workflows/build-docker-images-release.yml

```diff
@@ -21,44 +21,40 @@ jobs:
 
   version-cpu:
     name: "Latest Accelerate CPU [version]"
-    runs-on: ubuntu-latest
+    runs-on: [self-hosted, docker-gpu, multi-gpu]
```

Comment:
do we need to run on 2 different types of VM for building the image(s)?

Reply:
We do not, these are just all the tags for this particular machine.

Reply:
oh yeah, sorry. Get confused with the matrix thing 😅

Reply:
No worries at all, fully understandable :)